app/notification/application/README.md

```diff
@@ -17,12 +17,11 @@ To post data, use /send endpoint with POST method. Data must be in the following
 
     {
         "type": "APPLICATION_RECORD_OF_SUBMISSION",
-        "to": "testing_application@example.com",
+        "to": "email@example.com",
```

Comment:
Can we update the README with the other template names, or say where they can be found?

Reply:
Updated! 🙂